share/picongpu/tests/compileParticlePusher/include/picongpu/param/fileOutput.param

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2013-2020 Axel Huebl, Rene Widera, Felix Schmitt,
```

Comment:
If you define all output lists to be empty, why not remove this param file and avoid calling output in the cfg - it is a compile test anyway.

Reply:
You are right, I copied the other compile-time test suite without thinking of it. Guess were redundant there as well, but at least I will not add those in my PR.

Reply:
Now I think i understood the logic of other compile-time tests. By making the lists of what to output explicitly empty we guarantee the output part is not being compiled, regardless of whether openPMD-API / ADIOS dependencies were found. While with the standard `fileOutput.param` would still compile this part and thus spend compile-time for it, although this is not what we are testing in this suite.

Reply:
@sbastrakov Thanks for clarifying this. But this will result in a lot of code duplications. Would linking the duplicated files or a common base for all test resolve this issue?

Reply:
It is also true. I think the issue would become less severe while things are gradually moved out from `.param` files to `.cfg` files. Additionally, perhaps we could somehow explicitly specify build dependencies for tests? Then e.g. if I was positive there is no openPMD-API, ADIOS and ISAAC for sure, the corresponding files could have been safely deleted.